models/keyword_test.go

```diff
@@ -144,6 +144,35 @@ var _ = Describe("Keyword", func() {
 		})
 	})
 
+	Describe("#FindKeywordBy", func() {
+		Context("given the keyword is belong to the user", func() {
```

Comment:
While the method `FindKeywordBy` has been created, the description still somehow sounds like the previous method name `IeBelongTo` 🙈 Seems like a small type:

```suggestion
		Context("given the keyword belongs to the user", func() {
```